xml/docu_styleguide.terminology.xml

```diff
@@ -1820,11 +1847,27 @@
               <emphasis>kernel space</emphasis>
       </entry>
      </row>
+     <row>
+        <entry>utilization</entry>
+        <entry>utilisation [British]</entry>
+        <entry>noun; an act or instance of making practical or 
+                profitable use of something, especially in 
+                <emphasis>CPU utilization</emphasis>, 
+                <emphasis>memory utilization</emphasis>     
```

Comment:
This is the definition from dictionary.com. While it is certainly right, I don't think it goes together well with CPU or memory utilization. Unfortunately I don't have a better definition atm.